lib/allowed-features.js

```diff
@@ -49,9 +51,21 @@ export const FEATURE_FLAGS = {
 /**
  * Returns true if feature is allowed for this collective type, false otherwise.
  */
-export const isFeatureAllowedForCollectiveType = (collectiveType, feature) => {
-  const allowedTypes = FeatureAllowedForTypes[feature];
-  return allowedTypes ? allowedTypes.includes(collectiveType) : true;
+export const isFeatureAllowedForCollectiveType = (collective, feature) => {
```

Comment:
Name doesn't seem appropriate anymore

```suggestion
export const isFeatureAllowedForCollective = (collective, feature) => {
```